decoder/DecoderApi_unittest.cpp

```diff
@@ -330,6 +331,8 @@ TEST_P(DecodeApiTest, Format_Change)
     EXPECT_TRUE(inFrames > 0);
     EXPECT_EQ((int)formats.size(), 1);
     EXPECT_EQ(inFrames, outFrames);
+    delete allocator;
```

Comment:
allocator is a local variable. nobody will use it after this point. No need set NULL to it